SPECS/postgresql/postgresql.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=PostgreSQL database server
+Documentation=man:postgres(1)
+After=network-online.target multi-user.target
+Wants=network-online.target
+
+[Service]
+Type=notify
+User=postgres
+ExecStart=/usr/bin/postgres -D /usr/local/pgsql/data
+ExecReload=/bin/kill -HUP $MAINPID
+KillMode=mixed
+KillSignal=SIGINT
```

Comment:
Is this right?
https://www.postgresql.org/docs/16/server-shutdown.html

I would think Term is better